src/dfa/CM_Global.h

```diff
@@ -229,7 +229,7 @@ class Symtab
    static short    length;          // Current length of symbol name.
    static int      cell;            // Current hash cell number.
    static int      sti;             // Current symbol-table index.
-   static uint     hashdiv;         // Hash divisor.
+   static uint32     hashdiv;         // Hash divisor.
```

Comment:
Please explain why these changes are needed for mingw64.

Does that compiler produce windows executables?  (in other words, why use it?)

Reply:
Maybe I should use "unsigned int", yes it mingw compilers does generate windows executables.